app/main.js

```diff
@@ -70,15 +70,21 @@ app.on('ready', function() {
 
   // Create the browser window.
   mainWindow = new BrowserWindow({
+    backgroundColor: '#ffffff',
```

Comment:
According to the documentation, this is the default. Is this really needed?

And why is it here? Mind explaining the rest of the solution a bit?

Reply:
I think there might be a bug with electron in high sierra as reported here https://github.com/electron/electron/issues/10559 so the default colour turn grey and throw this error 
```
Electron[10948:1810625] *** WARNING: Textured window <AtomNSWindow: 0x7ff440c1af60> is getting an implicitly transparent titlebar. This will break when linking against newer SDKs. Use NSWindow's -titlebarAppearsTransparent=YES instead.
```
I solve it by setting the colour to white, might not be the best solution but at least it looks better? oh wait maybe i can look into the `-titlebarAppearsTransparent=YES` as shown in the error.

Reply:
> oh wait maybe i can look into the `-titlebarAppearsTransparent=YES` as shown in the error.

Please do, as the `ready-to-show` workaround is something I’d prefer to avoid.

Thank you for the explanation!

Reply:
I think at the moment can't really be fixed? Only solution is the fallback I made? Seems to be a known issue with electron and high Sierra :( https://github.com/electron/electron/issues/10521

btw there's this blue line on top, looks like a webkit highlight is that normal? tried to get rid of it but couldn't 
![screen shot 2017-09-29 at 10 09 28 am](https://user-images.githubusercontent.com/697155/30998018-4e27ef18-a4fe-11e7-9e37-2c70b582161e.png)

Reply:
> btw there's this blue line on top, looks like a webkit highlight is that normal?

It’s caused by [focusing the webview](https://github.com/vitorgalvao/fog/blob/master/app/main.js#L98L101), so we can use the keyboard shortcuts.